scripts/services/map_manager.gd

```diff
@@ -3,11 +3,17 @@ extends Node
 const MAX_MAP_SIZE = 40
 const LIST_FILE_PATH = "user://known_maps_list.json"
 const MAP_PATH = "user://"
+const BUNDLED_MAP_PATH = "res://assets/maps/"
 const MAP_EXTENSION = ".tofmap.json"
+const MAIN_MENU_MAP_NAME = "main_menu_bg"
```

Comment:
This will do for now, in the future it could be changed to an array, so we could add more names of special purpose maps if needed.